node.go

```diff
@@ -110,7 +137,7 @@ func (node *Node) handleGet(request Request) {
 			responses += 1
 		}
 		//TODO: consolidate vals
-		node.getReturn <- vals
+		node.getReturn <- consolidateDataVals(vals)
```

Comment:
Returns an array of Data vals with the highest clock values associated to their respective nodes